Content.Shared/_Impstation/CCVar/ImpCCVars.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Robust.Shared;
+using Robust.Shared.Configuration;
+
+namespace Content.Shared._Impstation.CCVar;
+
+// ReSharper disable once InconsistentNaming
+[CVarDefs]
+public sealed class ImpCCVars : CVars
+{
+    /// <summary>
+    /// The number of shared moods to give thaven by default.
+    /// </summary>
+    public static readonly CVarDef<uint> ThavenSharedMoodCount =
+        CVarDef.Create<uint>("thaven.shared_mood_count", 1, CVar.SERVERONLY);
```

Comment:
Should it be 1?
Isn't the funny thing about Thaven is all having the same shared moods?